python/Ganga/Core/GangaThread/WorkerThreads/__init__.py

```diff
@@ -2,6 +2,9 @@
 _global_queues = None
 _queues_interface = None
 
+def getQueues():
+    return _global_queues
+
```

Comment:
Yes but I am undecided if I want to check for if queues have started or not and issue a start command if they have not.
This can only be done from a getter I think.